Makefile

```diff
@@ -0,0 +1,26 @@
+ifeq ($(shell uname),Darwin)
+  EXT := dylib
+else
+  EXT := so
+endif
+
+source_files := $(wildcard src/*.rs)
+
+all: format lint build test
```

Comment:
We do have a Noxfile and hence I would recommend to add needed shortcuts to the Noxfile instead of adding a separate means of automation. (I do not really care whether Makefile or Noxfile, just that we use only one of it and not both. For a Python extension, a Noxfile does seem closer to home though...)

Reply:
Thanks for your explanation! I have removed the Makefile now